### PR TITLE
Adding Adopters

### PR DESCRIPTION
**Pull request by tylerhannan**
Adding numerous adopters.

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
This is an automated comment for commit ea0cf7b0117041075805ee65cbd7d153ae58f30b with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1272/ea0cf7b0117041075805ee65cbd7d153ae58f30b/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: docs/en/about-us/adopters.md [hunk 1]
```diff
@@ -227,6 +235,7 @@ The following list of companies using ClickHouse and their success stories is as
 | [QuickCheck](https://quickcheck.ng/) | FinTech | Analytics | — | — | [Blog post, May 2022](https://clickhouse.com/blog/how-quickcheck-uses-clickhouse-to-bring-banking-to-the-unbanked/) |
 | [R-Vision](https://rvision.pro/en/) | Information Security | — | — | — | [Article in Russian, December 2021](https://www.anti-malware.ru/reviews/R-Vision-SENSE-15) |
 | [Raiffeisenbank](https://www.rbinternational.com/) | Banking | Analytics | — | — | [Lecture in Russian, December 2020](https://cs.hse.ru/announcements/421965599.html) |
+| [Railway](https://railway.app/) | Software & Technology | PaaS Software Tools | — | — | [Blog, May 2023](https://railway.app/changelog/2023-05-19-horizontal-scaling) |
```
Comment (DanRoscigno) [inline docs/en/about-us/adopters.md:238]:
```suggestion
| [Railway](https://railway.app/) | Software & Technology | PaaS Software Tools | — | — | [Blog, May 2023](https://railway.app/changelog/2023-05-19-horizontal-scaling#logs-are-getting-faster) |
```

File: docs/en/about-us/adopters.md [hunk 2]
```diff
@@ -227,6 +235,7 @@ The following list of companies using ClickHouse and their success stories is as
 | [QuickCheck](https://quickcheck.ng/) | FinTech | Analytics | — | — | [Blog post, May 2022](https://clickhouse.com/blog/how-quickcheck-uses-clickhouse-to-bring-banking-to-the-unbanked/) |
 | [R-Vision](https://rvision.pro/en/) | Information Security | — | — | — | [Article in Russian, December 2021](https://www.anti-malware.ru/reviews/R-Vision-SENSE-15) |
 | [Raiffeisenbank](https://www.rbinternational.com/) | Banking | Analytics | — | — | [Lecture in Russian, December 2020](https://cs.hse.ru/announcements/421965599.html) |
+| [Railway](https://railway.app/) | Software & Technology | PaaS Software Tools | — | — | [Blog, May 2023](https://railway.app/changelog/2023-05-19-horizontal-scaling#logs-are-getting-faster) |
```
Comment (DanRoscigno) [inline docs/en/about-us/adopters.md:238]:
```suggestion
| [Railway](https://railway.app/) | Software & Technology | PaaS Software Tools | — | — | [Changelog, May 2023](https://railway.app/changelog/2023-05-19-horizontal-scaling#logs-are-getting-faster) |
```

**Review by DanRoscigno — APPROVED** [review]
Thanks so much!!!

**Pull request merged**